sklearn/cluster/tests/test_hierarchical.py

```diff
@@ -573,6 +574,21 @@ def test_agg_n_clusters():
         assert_raise_message(ValueError, msg, agc.fit, X)
 
 
+def test_agg_n_cluster_and_distance_threshold():
+    # Test that when distance_threshold is set that n_clusters is ignored
```

Comment:
I don't think this test really makes sense anymore now that both attributes are exclusive